documentation/projects/proposals/search_relevancy_sandbox/20230530-implementation_plan_staging_elasticsearch_reindex_dags.md

```diff
@@ -0,0 +1,192 @@
+# 2023-06-08 Implementation Plan: Staging Elasticsearch Reindex DAGs
+
+**Author**: @krysal
+
+## Reviewers
+
+- [ ] @AetherUnbound
+- [ ] @sarayourfriend
+
+## Project links
+
+- [Project Thread](https://github.com/WordPress/openverse/issues/392)
+- [Project Proposal](./20230331-project_proposal_search_relevancy_sandbox.md)
+
+## Overview
+
+This document describes the addition of two DAGs for Elasticsearch (ES) index
+creation ––full and proportional-by-provider–– which will allow us to decouple
+the process from the long Ingestion server's data refresh process and experiment
+with smaller indices. Also includes the adoption of two new index aliases for
+ease of handling and querying the new index types from the API with the
+[`internal__index`](api_ii_param) param.
+
+[api_ii_param]: https://github.com/WordPress/openverse/pull/2073
```

Comment:
It meant to be a [Reference-style link](https://www.markdownguide.org/basic-syntax#reference-style-links) but apparently I got it wrong 😅 Which is strange, I had always used it like this here in GH but maybe it's not an standard form 🤔

Reply:
I didn't know about that feature! It turns out MyST might just be using a different syntax: https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#markdown-link-syntax

Rather than `[`internal__index`](api_ii_param)` it would be `[`internal__index`][api_ii_param]`.

TIL about this feature!